### PR TITLE
feat(notes panel): add ui outline for notes panel

### PR DESCRIPTION
**Pull request by sidag95**
<NO MERGE>

<TODO Add Description>

**Summary**

**Test plan**

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by erezrokah** [comment]
Hi @sidag95, thank you for putting this up and sorry for the late follow.
I like the direction of the PR! How would you like to proceed? 
I think the tab order should be `Preview, Notes, i18n`, WDYT?
For the buttons we'll need to use the same styling as other CMS buttons, e.g. 
image

Because the way we ship the CMS (see https://github.com/netlify/netlify-cms/issues/4967 and https://github.com/netlify/netlify-cms/issues/2026) we'll need to class components and not react hooks.

**Comment by sidag95** [comment]
Hi @erezrokah currently I am looking at the data flow and how backends are hooked up with core. As the next step, I was thinking of hooking up Notes with the data store and github backend. Once the data flow is established, I am thinking of focusing on the frontend. So basically
- Create notes flow in the github backend
- Update UI to take data from the store
- Update the UX and UI
- Work on rest of the backends

> I think the tab order should be Preview, Notes, i18n, WDYT?


Yes this order makes sense. Will use this. Will also update the styling.

**Comment by erezrokah** [comment]
> * Create notes flow in the github backend
> * Update UI to take data from the store
> * Update the UX and UI
> * Work on rest of the backends

Sounds like a plan 🚀 

> P.S. The progress for this PR will be a little slow

No worries, completely understand. Thank you for leading it ❤️

File: packages/netlify-cms-backend-github/src/implementation.tsx [hunk 1]
```diff
@@ -670,4 +670,23 @@ export default class GitHub implements Implementation {
       'Failed to acquire publish entry lock',
     );
   }
+
+  async listNotes(collectionName: string, slug: string) {
+    const comments = await this.api!.readComments(collectionName, slug);
+    console.log(comments);
```
Comment (batuhankok) [inline packages/netlify-cms-backend-github/src/implementation.tsx:676]: Is it necessary to log the comments every time to the console?

**Comment by martinjagodic** [comment]
@sidag95 are you still interested in moving this forward?

**Comment by sidag95** [comment]
> @sidag95 are you still interested in moving this forward?

Hey @martinjagodic, at this moment, it would be difficult for me to proceed with this :/

**Pull request closed without merging**